x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/RemoteAccessHeaders.java

```diff
@@ -15,42 +15,42 @@
 
 public final class RemoteAccessHeaders {
 
-    public static final String REMOTE_ACCESS_CLUSTER_CREDENTIAL_HEADER_KEY = "_remote_access_cluster_credential";
-    private final String clusterCredentialsHeader;
+    public static final String REMOTE_CLUSTER_AUTHORIZATION_HEADER_KEY = "_remote_cluster_authorization";
```

Comment:
This is a "breaking change" but we're behind a feature flag so I don't see a reason not make it.